src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3926,6 +3926,133 @@ def test_aks_update_with_workload_identity(self, resource_group, resource_group_
             self.check('securityProfile.workloadIdentity.enabled', False),
         ])
 
+    @live_only()
```

Comment:
May I ask why those new tests are `@live_only()`

Reply:
Ah, the backend service is not ready, so the tests (also the new options) could not work. The RP release is in progress and should be available soon.